dockers/Screenshotter/README.md

```diff
@@ -21,7 +21,7 @@ docker containers with the selenium setups running.  Essentially you
 are encouraged to reproduce the steps from `screenshotter.sh`
 manually.  Example run for Firefox:
 
-    container=$(docker run -d -P selenium/standalone-firefox:2.46.0)
+    container=$(docker run -d -P selenium/standalone-firefox:2.48.2)
```

Comment:
I wondered about that at first, too. But it turns out that's what the `screenshotter.sh` script is using. So this updates the README to match the automated generation. More consistent that way.